src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonName.java

```diff
@@ -0,0 +1,128 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Example PAT+RAT56:OBI+++SU:KENNEDY+FO:SARAH+TI:MISS+MI:ANGELA'
+ */
+@Setter
+@Builder
+public class PersonName extends Segment {
+
+    public static final String KEY = "PNA";
+    public static final String QUALIFIER = "PAT";
+    public static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+
+    //all properties are optional
+    private String nhsNumber;
```

Comment:
Missing getter?
Add `@Getter` on class instead on each field

Reply:
nhsNumber has not-standard getter 'getNhsNumber'

Reply:
Rename the static `getNhsNumber` to `extractNhsNumber` so that the getter can be defined as expected